include/bluetooth/direction.h

```diff
@@ -217,4 +230,22 @@ int bt_df_conn_cte_rx_enable(struct bt_conn *conn, const struct bt_df_conn_cte_r
  */
 int bt_df_conn_cte_rx_disable(struct bt_conn *conn);
 
+/**
+ * @brief Set Constant Tone Extension transmission parameters for a connection.
+ *
+ * The function is available if @kconfig{CONFIG_BT_DF_CONNECTION_CTE_TX} is enabled.
+ *
+ * @note @p params contents may not be completely validated and forwarded to Controller.
+ * If bt_df_conn_cte_tx_param.cte_types bits BT_DF_CTE_TYPE_AOD_1US and BT_DF_CTE_TYPE_AOD_2US are
+ * not set, the function will not validate members bt_df_conn_cte_tx_param.num_ant_ids and
+ * bt_df_conn_cte_tx_param.ant_ids. Also these members of @p params will not be forwarded to
+ * Controller.
```

Comment:
Not sure if it is better, but have you considered adding this information for the parameter documentation itself (i.e. move it to line 245)?

Reply:
Or perhaps in the struct documentation instead.

Reply:
In my humble opinion it should be part of a function documentation because it is function behavior.
I didn't put it in parameter description but as a note to avoid long extended description of an argument. 
If you strongly advise to move it to line 245 then I can do it.

Reply:
It is fine, if this is behavior specific to this function, and not the parameter struct :) It was not as much of a request, but rather a question if you had considered it.